packages/terra-slide-panel-manager/tests/wdio/slide-panel-manager-spec.js

```diff
@@ -191,96 +196,108 @@ Terra.describeViewports('SlidePanelManager - Managed Header', ['large'], () => {
   const selector = '#root-component';
 
   describe('Disclosure - Header Close', () => {
-    before(() => browser.url('/#/raw/tests/terra-slide-panel-manager/slide-panel-manager/slide-panel-manager-with-header-adapter').refresh());
+    before(() => {
+      browser.url('/#/raw/tests/terra-slide-panel-manager/slide-panel-manager/slide-panel-manager-with-header-adapter');
+      browser.refresh();
```

Comment:
you can't chain these anymore?

Reply:
Nope. Command chaining was removed in wdio v5.

Reply:
![](https://media.giphy.com/media/l22ysLe54hZP0wubek/giphy.gif)